xml/art_sle_ha_pmremote.xml

```diff
@@ -274,50 +301,213 @@
+         be helpful to find connection, authentication, or configuration
+         problems. Multiple <option>-v</option> options increase the verbosity.
+        </para>
+       </listitem>
+      </varlistentry>
+     </variablelist>
+     <para>If needed, add more guest nodes and configure them as described above.</para>
+    </step>
+    <step>
+     <para>
+      Shut down the KVM guest and proceed with <xref
+       linkend="sec.ha.pmremote.config.uc1.integrate"/>.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec.ha.pmremote.config.uc1.integrate">
+   <title>Integrating KVM Guests into Cluster</title>
+   <para>To integrate the KVM guests into the cluster, perform the following
+    steps on the physical host <systemitem class="domainname">&node1;</systemitem>:</para>
+   <procedure>
+    <step>
+     <para>Start &pace;:</para>
+     <screen>&prompt.root;<command>systemctl</command> start pacemaker</screen>
+    </step>
+    <step>
+     <para>Check if your KVM guest node and IP address is listed
+     in the file <filename>/etc/hosts</filename>.</para>
+    </step>
+    <step>
+     <para>
+      Create a <systemitem>ocf:pacemaker:remote</systemitem> primitive:
+     </para>
+     <remark>toms 2017-07-11: Check the interval values</remark>
+     <screen>&prompt.root;<command>crm</command> configure
+&prompt.crm.conf;<command>primitive</command> &node3; ocf:pacemaker:remote \
+     params server=&node3; recommection_interval=10 \
+     op monitor interval=
+&prompt.crm.conf;<command>commit</command></screen>
+    </step>
+    <step>
+     <para>Check the status of the cluster with the command
+      <command>crm status</command>. It
+      should contain a running cluster with nodes that are all accessible.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="sec.ha.pmremote.install.virt-guest-nodes">
+  <title>Setting Up a Cluster with Virtual Guest Nodes</title>
+  <remark>toms 2017-07-11: Are other virtualization techniques supported, like
+  XEN...?</remark>
+  <para>
+   Before you proceed, install and set up a basic two-node cluster. This task
+   is described in <citetitle>&instquick;</citetitle>. This will lead to a
+   two-node cluster with the two physical hosts <systemitem
+    class="domainname">&node1;</systemitem> and <systemitem
+     class="domainname">&node2;</systemitem>.
+  </para>
+  <para>In the following example setup, KVM is used for setting up the virtual
+   guest nodes:</para>
+  <orderedlist>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.config.uc2.phys.host"/></para>
+   </listitem>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.config.uc2.kvm.guest"/></para>
+   </listitem>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.integrate.uc2.guestsintocluster"/></para>
+   </listitem>
+  </orderedlist>
+
+  <sect2 xml:id="sec.ha.pmremote.config.uc2.phys.host">
+   <title>Setting up a Corosync Cluster with Physical Hosts</title>
```

Comment:
Same comment as for 3.1:
_'Configuring Corosync Cluster Nodes' is very general. What we show here is more 'enhance cluster nodes by creating authkey to support pacemaker_remote'. Wouldn't it be better to add some more specific title, like 'Configuring Authentication Key'?_